management-ui/server.py

```diff
@@ -30,19 +35,41 @@
 from rt_m1_client.data_store import JSONFileDataStore
 from rt_m1_client.exceptions import M1Error
 
+from rt_media_configuration.media_configuration import MediaConfiguration
+from rt_media_configuration.importers.m1_importer import M1SessionImporter
+from rt_media_configuration.media_entry import MediaEntry
+from rt_media_configuration.media_distribution import MediaDistribution
+from rt_media_configuration.media_entry_point import MediaEntryPoint
+from rt_media_configuration.media_app_distribution import MediaAppDistribution
+
 config = Configuration()
```

Comment:
There's a singleton Configuration object already available, might be better to use that:
```python
from rt_m1_client import app_configuration

config = app_configuration
```